core/src/main/java/com/google/zxing/qrcode/encoder/Encoder.java

```diff
@@ -49,6 +49,8 @@
       25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, -1, -1, -1, -1, -1,  // 0x50-0x5f
   };
 
+//@Sean: Is the default encoding indeed ISO-8859-1? I also implemented the MinimalEncoder assuming this but now I looked at the specification and find
```

Comment:
That's what 6.4.5 in the 2006 spec says. Which version are you looking at? I don't see an 8.3.1 in my copy.
In any event I wouldn't change this behavior.

Reply:
Good, mine is older. Is there a way that I can get your copy?

Reply:
I don't believe copies of the ISO spec are supposed to be shared as they are, weirdly, for sale - if it comes up somehow I can provide some info from it

Reply:
Thanks, that's very good. Something else, there seem to be different opinions on who presses "Resolve conversation", the request opener or the reviewer. The git hub documentation says the request opener so that I will now mark those things as resolved that I think are resolved. If you handle it differently, please tell me.